packages/docs/stories/components/input.stories.ts

```diff
@@ -97,6 +97,28 @@ export const ReadonlyInputs: Story = {
   },
 };
 
+/** The focus attribute provides feedback to the users, informing them that the input component is ready for use.  */
```

Comment:
It's not an attribute. :) Please rephrase, maybe together with @LeaLeRose so she can provide it in Figma as well.

Reply:
Yeah, I wasn't sure what to write

Reply:
the documentary is now a lot more than the things we provide in Figma or Shoelace? Is this planned? do I have to change this as well?